tests/main.sh

```diff
@@ -79,6 +79,8 @@ RUNTEST() {
 	fi
 }
 
+TESTCASE"Recursive Walk"
+for i in `seq 1 100`; ls -alR "${LXCFSDIR}" &> /dev/null; done
```

Comment:
`&>` is a bashism, the non-bash variant is `>/dev/null 2>&1`

Reply:
thanks!